diff --git a/core/board_config.mk b/core/board_config.mk
index ea6ec1e4ec..0254994040 100644
--- a/core/board_config.mk
+++ b/core/board_config.mk
@@ -149,7 +149,7 @@ _board_strip_readonly_list += $(_build_broken_var_list) \
 
 # Conditional to building on linux, as dex2oat currently does not work on darwin.
 ifeq ($(HOST_OS),linux)
-  WITH_DEXPREOPT := true
+  WITH_DEXPREOPT ?= true
 endif
 
 # ###############################################################
